dialogue-pizzaconundrum.js

```diff
@@ -34,7 +34,7 @@ const PIZZAMOOD_SAD = [
   {
     suspicion: ANYSUS,
     segments: [
-      "hes: #GiveMAriaAHug 🫂❤️",
+      "hes: #GiveMAriaAHug ❤️",
       "bibop: sad story... 😿",
       "thelibrarian: No pizza for the brave ☹️",
       "hes: Poor Maria 🌧️☹️☁️",
```

Comment:
Just checking, are these frowny faces intentional? This is how I see it:
![Screen Shot 2022-02-05 at 11 32 36 AM](https://user-images.githubusercontent.com/21364548/152650417-75d57e4f-f418-42bb-beb5-e498fc27b7a9.png)

Reply:
Yes, that's the page for it: https://emojipedia.org/frowning-face/

Reply:
That page shows an emoji, but in the screenshot it's just an outline. Which is correct?